server/src/database/db.ts

```diff
@@ -10,7 +10,7 @@ class Database {
 
     constructor() {
         // If on docker pool should take config, if on local database pool should take localconfig
-        this.pool = new Pool(config);
+        this.pool = new Pool((process.env.DB_ENV === "local") ? localconfig : config);
```

Comment:
Also for whatever reason, this is returning false even if DB_ENV is set. So what worked for me was `process.env.DB_ENV ? ` just to check that the variable exists.

Reply:
In that case, should we rename it as `DB_LOCAL` to indicate that we're working locally?

Reply:
Yes good suggestion @yehee. Did you test it? Im assuming the current setup works since you're on Mac.

Reply:
Hmm, I can't seem to replicate that behaviour? What did you run to get it to be false?

Reply:
![dbEnv](https://user-images.githubusercontent.com/42220961/99895165-65a67600-2c3c-11eb-9ae1-4aadbeab6a65.png)

Reply:
Maybe my PC needs a restart lol ^ ... I have no idea why the above screenshot is happening.

Reply:
@johnagl can't test it locally since I don't have postgres on my machine, I might to do something for those who are working with docker!

Reply:
I'm totally lost LOL it works perfectly fine on my machine haha do you mind clarifying your suggestion about `process.env.DB_ENV?`  ?

I just pushed changes to use `DB_LOCAL=true` so hopefully that works.

Reply:
Works now but I still error on the current script, need to use:  `"start:localwindows": "set DB_ENV=local && ts-node-dev --respawn src/index.ts"`

Reply:
We can keep the original script but i think certain windows 10 users need the one im using.

Reply:
Oh hmm, does it only work if you use the new script you pasted? I'll push changes for that anyway and I guess people can choose which one they run

Reply:
Yeah thats what i meant. My script is for a specific subset of windows 10 users. Your original one works fine for Linux/Mac, and apparently your laptop - whatever it is I dont know if you're on windows, linux, mac or a hybrid of the 3.